webapp/pages/profile/_id/_slug.vue

```diff
@@ -67,7 +69,7 @@
           <ds-space margin="small">
             <template v-if="!myProfile">
               <hc-follow-button
-                v-if="!user.isMuted"
+                v-if="!user.isMuted || !user.isBlocked"
```

Comment:
This is equivalent to:
```js
!(user.isMuted && user.isBlocked)
```

Are you sure? This whould show the button e.g. if you muted the user but did not block the user. :confused: 

Either:
```js
!(user.isMuted || user.isBlocked)
```

Or just leave it as it was:
```js
!user.isMuted
```
in case you want to allow that you can block **and** follow a user at the same time.

Reply:
see my comment :point_up_2: